.github/workflows/tests.yml

```
@@ -329,16 +331,29 @@ jobs:
         if: ${{ github.event_name == 'schedule' }}
         run: echo "NOT_MARKERS=$NOT_MARKERS refcount" >> $GITHUB_ENV
 
+      - name: Export GitHub Actions cache environment variables
```

This is obscure enough to deserve a comment.
```suggestion
      - name: Export GitHub Actions cache environment variables
      # https://learn.microsoft.com/en-us/vcpkg/consume/binary-caching-github-actions-cache
```